ios/DevStack.xcodeproj/project.pbxproj

```diff
@@ -581,6 +583,42 @@
 			shellPath = /bin/sh;
 			shellScript = "\"${BUILD_DIR%/Build/*}/SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/run\"\n";
 		};
+		A327766E2BAD960400CC8CF4 /* Generate error messages */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Generate error messages";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "cd \"${PROJECT_DIR}\"\nexec > prebuild-error-messages.log 2>&1 # For debugging\nsh scripts/generate-error-messages.sh\n";
```

Comment:
We can get rid of `exec > prebuild-error-messages.log 2>&1 # For debugging` line - we needed it when we called the script from pre-actions (in schemes), because the output wasn't visible anywhere in Xcode. But now (calling from `BuildKMP` dummy framework) it is visible directly in the Xcode, so external log is not needed. 🙂